eslint.config.mjs

```diff
@@ -7,12 +7,15 @@ import eslintPluginMocha from 'eslint-plugin-mocha'
 import eslintPluginN from 'eslint-plugin-n'
 import eslintPluginPromise from 'eslint-plugin-promise'
 import eslintPluginUnicorn from 'eslint-plugin-unicorn'
+import { readFileSync } from 'fs'
 import globals from 'globals'
 
 import eslintProcessEnv from './eslint-rules/eslint-process-env.mjs'
 import eslintEnvAliases from './eslint-rules/eslint-env-aliases.mjs'
 import eslintSafeTypeOfObject from './eslint-rules/eslint-safe-typeof-object.mjs'
 
+const { dependencies } = JSON.parse(readFileSync('./vendor/package.json', 'utf8'))
```

Comment:
Why not this?

```suggestion
const { dependencies } = require('./vendor/package.json')
```

Reply:
Looking at how this is used below, I'd actually extend it to be simply:

```suggestion
const allowModules = Object.keys(require('./vendor/package.json').dependencies)
```

And then update the references below

Reply:
Using require in ESM can be a bit complicated 😅 It doesn't "just work", I'd have to create an instance of it through `Module`. I figured I'd just stick to ESM best practices instead.